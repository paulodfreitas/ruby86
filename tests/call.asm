.module m0
.pseg
	irmovl $100,%esp
	irmovl $5, %eax
	irmovl $10, %ebx
	call my_func
	irmovl $3, %ebx
	call my_func
	halt

my_func:
    addl %ebx, %eax
    ret
.end
