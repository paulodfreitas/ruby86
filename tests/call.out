@ 0000
30
84
00
01
00
00
30
80
05
00
00
00
30
83
10
00
00
00
80
23
00
00
00
30
83
03
00
00
00
80
23
00
00
00
10
60
30
90
@ 0026
